ios/kmbuild.sh

```diff
@@ -210,11 +211,18 @@ update_bundle ( ) {
 update_bundle
 
 if [ $DO_CARTHAGE = true ]; then
-    echo
-    echo "Load dependencies with Carthage"
-    # TODO: Replace the workaround-script with the base `carthage` command once
-    # we've properly updated to 0.37's better approach.
-    $KEYMAN_ROOT/resources/build/carthage-workaround.sh bootstrap --platform iOS || fail "carthage boostrap failed"
+  echo
+  echo "Load dependencies with Carthage"
+
+  carthage checkout || fail "Carthage dependency loading failed"
+
+  # Carthage sometimes picks the wrong .xcworkspace if two are available in a dependency's repo.
+  # Easiest way to override it - delete the wrong one (or just its scheme)
+
+  # Deleted workspace - a test for proper deployment to CocoaPods.  Doesn't matter here.
+  rm -r ./Carthage/Checkouts/DeviceKit/CocoaPodsVerification/ || fail "Carthage dependency loading failed"
```

Comment:
- [ ] This will need to be ported to stable-14.0 once we update Carthage on the build agents.

I just noticed the same behavior this addresses while working on a different PR, so it's independent of `--use-frameworks` - it's a Carthage 0.38.0 bug.  Well... that or an 'intended behavior' redefinition.

Reply:
This won't affect the configuration of the stable build.  It's just a mild script tweak to maintain the status quo.